### DISCUSSION TITLE
MilvusClient 创建新的database

### DISCUSSION THREAD

**Discussion by gggdroa** [milvus-io/milvus#33779] [Show and tell]
MilvusClient 没有create_database函数吗？怎么构建新的database

**Comment by Reidddddd** [comment]
https://milvus.io/api-reference/pymilvus/v2.4.x/ORM/db/create_database.md

**Reply by gggdroa** [reply]
这个是知道的，他们还需要交叉使用吗？MilvusClient 这个没有直接的create哇？

**Reply by yhmo** [reply]
按理说应该要有，估计后续版本会加

PS: 以后请尽量用英文提问
